containers/ddev-webserver/ddev-webserver-base-scripts/start.sh

```diff
@@ -78,8 +78,17 @@ ls /var/www/html >/dev/null || (echo "/var/www/html does not seem to be healthy/
 # Make sure the TERMINUS_CACHE_DIR (/mnt/ddev-global-cache/terminus/cache) exists
 sudo mkdir -p ${TERMINUS_CACHE_DIR}
 
-sudo mkdir -p /mnt/ddev-global-cache/{bashhistory/${HOSTNAME},mysqlhistory/${HOSTNAME},nvm_dir/${HOSTNAME},npm/${HOSTNAME},yarn/${HOSTNAME}}
+sudo mkdir -p /mnt/ddev-global-cache/{bashhistory/${HOSTNAME},mysqlhistory/${HOSTNAME},nvm_dir/${HOSTNAME},npm,yarn}}
 sudo chown -R "$(id -u):$(id -g)" /mnt/ddev-global-cache/ /var/lib/php
+# The following ensures a persistent and shared "global" cache for
+# yarn1 (classic) and yarn2 (berry). In the case of yarn2, the global cache
+# will only be used if the project is configured to use it through it's won
+# enableGlobalCache configuration option. Assumes ~/.yarn/berry as the default
+# global folder.
+(cd && yarn config set cache-folder /mnt/ddev-global-cache/yarn || true)
```

Comment:
I think this will leave us in the homedir and could affect (future) additions below here. 
```suggestion
(pushd ~ && yarn config set cache-folder /mnt/ddev-global-cache/yarn || true)
popd
```

Reply:
the parenthesis makes it a sub shell command, it's run outside of the scope of the shell

Reply:
I took the idea from https://stackoverflow.com/questions/10382141/temporarily-change-current-working-directory-in-bash-to-run-a-command

Reply:
Fixed the type on the prod script too, is that what you meant with prod dockerfile?